chunks/lang-go/chunk.yaml

```diff
@@ -2,3 +2,6 @@ variants:
   - name: "1.18.0"
```

Comment:
Let us get rid of 1.18.0 as this won't be published with the change any more

Reply:
I disagree. We should be able to select versions. Getting rid of a chunk removes that option.

Reply:
Maybe I am misunderstanding this, but we are not publishing this variant anywhere after this change. In past the variants piled up quite a bit causing frequent timeouts during build which we cleaned up https://github.com/gitpod-io/workspace-images/pull/761/files

Reply:
@princerachit the new chunk will be done in `dazzle build`, which is generally quick.

We'll only use one of the two `go` chunk, 1.18.1, when we `dazzle combine`. Since the # of combinations isn't increasing, we will be less likely to timeout. If this were a new combination - I'd be worried - but it isn't, so I think we're okay.

Even if we do have a timeout, we can just run the build again (short term hack), and it'll more likely work.